README.adoc

```diff
@@ -130,6 +130,7 @@ When converting to HTML, images must be copied to the output location so that th
     <resources>
 ----
 outputDirectory:: defaults to `${project.build.directory}/generated-docs`
+outputFile:: defaults to `null`, can be used to override the name of the generated output file. This is useful for backends, that create a single file, e.g. the pdf backend. All output will be redirected to the same file, the same way as the `-o, --out-file=OUT_FILE` option from the `asciidoc` CLI command.
```

Comment:
Yes, have fixed it. But I just checked, the asciidoc command supports the same parameter, so I wasn't wrong 😁